### PR TITLE
{geo}[foss/2019a] GMT v5.4.5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)
Minor update to GMT for foss/2019a with same dependecy list as previous easyconfig.

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f4285232b5ebddbe6b0c0c99bb2d7820 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node321.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/34751ed66fdef68e9319cdc8b160ff8e for a full test report.

File: easybuild/easyconfigs/g/GMT/GMT-5.4.5-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+]
+
+separate_build_dir = True
+
+configopts = '-DCOPY_GSHHG=TRUE -DGSHHG_ROOT=%%(builddir)s/gshhg-gmt-%s ' % local_gshhg_ver
+configopts += '-DCOPY_DCW=TRUE -DDCW_ROOT=%%(builddir)s/dcw-gmt-%s ' % local_dcw_ver
+
+sanity_check_paths = {
+    'files': [
+        ['bin/%s' % b for b in ['gmt', 'isogmt', 'gmtswitch', 'gmt_shell_functions.sh']],
```
Comment (smoors) [inline easybuild/easyconfigs/g/GMT/GMT-5.4.5-foss-2019a.eb]: @lexming I am not sure it is ok to have a list in a list here. although the test succeeded, in the log I see that it checks for the first file only, `bin/gmt`.

Comment (boegel) [reply easybuild/easyconfigs/g/GMT/GMT-5.4.5-foss-2019a.eb]: @smoors Indeed, this isn't how it should be:

```python
sanity_check_paths = {
    'files': ['bin/%s' % b for b in ['gmt', 'isogmt', 'gmtswitch', 'gmt_shell_functions.sh']] +
             ['lib64/libgmt.%s.%s' % (SHLIB_EXT, version)],
    'dirs': ['include/%(namelower)s', 'share']
}

Comment (lexming) [reply easybuild/easyconfigs/g/GMT/GMT-5.4.5-foss-2019a.eb]: Thanks for the feedback. Fixed.

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**